src/drivers/ms5611/ms5611.cpp

```diff
@@ -98,8 +98,12 @@ enum MS5611_BUS {
  * MS5611/MS5607 internal constants and data structures.
  */
 
-/* internal conversion time: 9.17 ms, so should not be read at rates higher than 100 Hz */
-#define MS5611_CONVERSION_INTERVAL	25000	/* microseconds */
+/*
+ * Maximum internal conversion time for OSR 1024 is 2.28 ms. We set an update
+ * rate of 100Hz which is be very safe not to read the ADC before the
+ * conversion finished
+ */
+#define MS5611_CONVERSION_INTERVAL	10000	/* microseconds */
```

Comment:
It doesn't change the internal sensor configuration. We will just ask the sensor more frequently to get us a sample.

In the past we changed the OSR from 4096 to 1024 due to the self heating issue. However we kept the sampling rate which means we get more noisy data.  With OSR at 1024 each internal sensor conversion takes around 3msec. Getting a sample every 10msec appeared pretty safe and improved AltHold on my tests with MS5607